variables.tf

```diff
@@ -56,12 +56,23 @@ variable "ecr_scan_on_push" {
   description = "Controls if ECR should scan images after pushed."
 }
 
-variable "ecr_number_of_images_to_keep" {
+variable "ecr_number_of_images_to_keep_on_main" {
   type        = number
   default     = 30
   description = "Controls how many images should be kept in the ECR repo."
 }
 
+variable "ecr_number_of_images_to_keep_on_branches" {
```

Comment:
Should we change the variable name to `ecr_number_of_images_from_branches` to be more concise and a bit shorter?